source/views/sis/course-search/row.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import * as React from 'react'
+import {Text} from 'react-native'
+import type {CourseType} from '../../../lib/course-search/types'
+import {ListRow, Title} from '../../components/list'
+import {Row} from '../../components/layout'
+import {FormattedLine} from '../components/formatted-line'
+
+type Props = {
+	course: CourseType,
+	onPress: CourseType => any,
+}
+
+export class CourseRow extends React.PureComponent<Props> {
+	onPress = () => {
+		this.props.onPress(this.props.course)
+	}
+
+	render() {
+		const {course} = this.props
+		return (
+			<ListRow arrowPosition="center" onPress={this.onPress}>
+				<Row>
+					<Title lines={1}>
+						<Text>{course.name}</Text>
```

Comment:
I believe that Title acts like Text in that you don't need the Text to wrap the raw string